### DISCUSSION TITLE
Function Test - Help needed?

### DISCUSSION THREAD

**Discussion by stefanpausch** [pmgducati/Blue-Retro-AIO-Units#35] [General]
I am about to order / build the project myself (Will take some time until all parts are here) and i can help with function tests if needed. 

Let me know if i can be any help and how to do a function test.

- NeoGeo, SuperGun, Jamma (consolized MVS, MVS with direct Ports, Jamma available)
- NES (PAL) (currently don't own one, but i know a few ppl who own one)
- Genesis / MegaDrive (Owning MegaDrive 1/2 and Analogue Version)
- Super Famicon / SNES NTSC /SNES PAL (All Version including Analogue One available)
- PSX (Is this Playstation 1? - Available in different country versions)
- Dreamcast (Available)

**Comment by pmgducati** [comment]
That would be great if you are willing to help, there are issues for each system that needs to be tested, you can put your findings there.  Also, PSX is PS1
